xds/src/main/java/io/grpc/xds/RbacFilter.java

```diff
@@ -93,9 +96,8 @@
   }
 
   @VisibleForTesting
-  @Nullable
   static ConfigOrError<RbacConfig> parseRbacConfig(RBAC rbac) {
-    // https://github.com/envoyproxy/envoy/blob/a12869fa9e9add4301a700978d5489e6a0cc0526/api/envoy/extensions/filters/http/rbac/v3/rbac.proto#L25
+    checkNotNull(rbac, "rbac");
```

Comment:
Well, this doesn't seem to necessary. My personal practice to decide if a `checkNotNull()` is needed are mostly based on (with decreasing strength of argument):
- Is the object saved and used later, so that the first place that causes NPE can be far from where the null object is received (aka, where is problem begins)? As the primary purpose for using this check is to determine the root cause of the NPE bug. If NPE happens right at the place the null object is received, getting the NPE stacktrace is equally good as failing the `checkNotNull` check (except `checkNotNull` gives the string message). A good example is for constructor arguments: arguments are saved and used later, so we want know the first place that an expected non-null variable is null instead of the place that the object is referenced.
- By convention, arguments not explicitly annotated with `@Nullable` are expected to be non-null.

Anyway, not a big issue...